tests/full-disk-encryption.spec.ts

```diff
@@ -27,8 +28,8 @@ test.describe('The main page', () => {
         await storagePage.validateEncryptionIsUsed();
         await storagePage.back();
 
-        await expect(page.getByText("SUSE ALP Dolomite")).toBeVisible({ timeout: 2 * minute });
-        await page.getByRole('link', { name: 'Users' }).click();
+        await expect(page.getByText(process.env.PRODUCTNAME)).toBeVisible({ timeout: 2 * minute });
```

Comment:
it looks like we don't need these sync point to check the title and know that we are in main page, we could just click on install and if we would not be there then it would be some error, so probably are not needed this one and the following, could you please check that?

Reply:
Not needed, removed. Updated VRs.